src/core/tests/Test_PatchInstaller.py

```diff
@@ -270,11 +270,11 @@ def test_stopwatch_properties_patch_install_fail(self):
     def test_write_installer_perf_logs_catch_exception(self):
```

Comment:
nit: Change the name of this method accordingly.

Reply:
Yeah, change it to reflect what the test is actually doing

Reply:
Renamed.